src/chains/ethereum/ethereum/tests/forking/cache/arbitraries.ts

```diff
@@ -112,7 +112,7 @@ export const Networks = (): fc.Arbitrary<Model> =>
     .tuple(
       Commands.AddNetwork(),
       fc.array(
-        fc.frequency(
+        fc.oneof(
```

Comment:
this was because I updated fast-check to a new major version and `frequency` was removed.